### PR TITLE
Dev: add BBD support; Closes #120

### LINKED ISSUE #120: Missing Battlebond BBD for use with cube
Unable to start a room with a cube list with two BBD cards.

### PR DESCRIPTION
**Pull request by HerveH44**
Add BBD cards. 

Though we don't should the set as draftable (it should be only for Two-Headed Giant)

### CODE REVIEW

**Comment by ZeldaZach** [comment]
If people want to draft it for chaos or smth, let em :)

**Comment by tooomm** [comment]
I was thinking the same. It doesn't really hurt to list them I guess.
Which actually applies to commander sets or all sets in general...

File: src/make/cards.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var setsToIgnore = ['TSB','ITP','CP1','CP2','CP3'];
 before()
 
 var types = ['core', 'expansion', 'commander', 'planechase', 'starter', 'un']
-var codes = ['EMA', 'MMA', 'VMA', 'CNS', 'TPR', 'MM2', 'EXP', 'MPS', 'CN2', 'MM3', 'MPS_AKH', "IMA"]
+var codes = ['EMA', 'MMA', 'VMA', 'CNS', 'TPR', 'MM2', 'EXP', 'MPS', 'CN2', 'MM3', 'MPS_AKH', "IMA", "BBD"]
```
Comment (tooomm) [inline src/make/cards.js]: Wondering why masters sets are not filtered by `masters` type, but some are listed by their codes instead?!

**Comment by tooomm** [comment]
What are you thoughts @HerveH44 ?

- https://github.com/dr4fters/dr4ft/pull/138#issuecomment-411874992

- https://github.com/dr4fters/dr4ft/pull/138#discussion_r209069676

**Comment by tooomm** [comment]
Superseded by #169

**Pull request closed without merging**